extensions/amp-analytics/0.1/test/test-analytics-root.js

```diff
@@ -641,6 +683,49 @@ describes.realWin(
       });
     });
 
+    describe('get amp elements', () => {
+      let child2;
+      let elements;
+
+      beforeEach(() => {
+        child2 = win.document.createElement('child');
+        body.appendChild(child2);
+        child.classList.add('i-amphtml-element');
+        child2.classList.add('i-amphtml-element');
+      });
+
+      it('should find all elements by selector', async () => {
```

Comment:
could you try one thing, add another element to the embed host and make sure it won't be selected.

Reply:
Ping on this comment : )

Reply:
@zhouyx Do you mean add another element that is non-amp?

Reply:
I meant add element with the className to the env.parentWin and then verify that they don't get selected. 
https://github.com/ampproject/amphtml/blob/master/testing/describes.js#L805

Reply:
Got it! Thanks for the explanation. Added an element called parentChild that is attached to the the parentWin.

Reply:
Looks great! Thanks